### PR TITLE
Updated docker-alfresco to 4.1.2 and fixed developer-documentation (g…

### PR DESCRIPTION
**Pull request by Ranecdev**
…enerate-doc) with correct hub reference

Fixes https://xenitsupport.jira.com/browse/ALFREDAPI-402

- [x] Is [CHANGELOG.md](https://github.com/xenit-eu/alfred-api/blob/master/CHANGELOG.md) extended?
- [ ] Does this PR avoid breaking the API? 
    Breaking changes include adding, changing or removing endpoints and/or JSON objects used in requests and responses.
- [x] Does the PR comply to REST HTTP result codes policy outlined in the [developer guide](https://github.com/xenit-eu/alfred-api/blob/master/developer-documentation)?
- [ ] Does the PR follow our [coding styleguide and other active procedures](https://xenitsupport.jira.com/wiki/spaces/XEN/pages/624558081/XeniT+Enhancement+Proposals+XEP)?
- [x] Is usage of `this.` prefix avoided?

See [README.md](./README.md) for full explanation.

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-Fixes https://xenitsupport.jira.com/browse/ALFREDAPI-
```
Comment (WimCrols) [inline .github/pull_request_template.md]: I just noticed this render to Italic. Can you make it bold (use 2 stars ** https://help.github.com/en/github/writing-on-github/basic-writing-and-formatting-syntax#styling-text)

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,10 +3,11 @@
 ## 2.3.0 (Unreleased)
 
 ## Changed
+* [ALFREDAPI-368](https://xenitsupport.jira.com/browse/ALFREDAPI-368): Replace obsoleted Gradle plugins (org.dm.bundle and ampde)
 * [ALFREDAPI-385](https://xenitsupport.jira.com/browse/ALFREDAPI-385): Change docker & compose files for integration tests to use harbor and docker.io/xenit
+* [ALFREDAPI-386](https://xenitsupport.jira.com/browse/ALFREDAPI-386): Upgrade to Gradle 5
 * [ALFREDAPI-387](https://xenitsupport.jira.com/browse/ALFREDAPI-387): Add existence check and 404 to working copies endpoint
-* [ALFREDAPI-368](https://xenitsupport.jira.com/browse/ALFREDAPI-368): Replace obsoleted Gradle plugins (org.dm.bundle and ampde)
-* [ALFREDAPI-386](https://xenitsupport.jira.com/browse/ALFREDAPI-386): Upgrade to Gradle 5 
+* [ALFREDAPI-402](https://xenitsupport.jira.com/browse/ALFREDAPI-402): Removed obsolete webscript (rest-v0.categories.CategoryGetWebscript)
```
Comment (WimCrols) [inline CHANGELOG.md]: Please put this under a subsection `### Deleted` below

File: CHANGELOG.md [hunk 3]
```diff
@@ -3,10 +3,11 @@
 ## 2.3.0 (Unreleased)
 
 ## Changed
+* [ALFREDAPI-368](https://xenitsupport.jira.com/browse/ALFREDAPI-368): Replace obsoleted Gradle plugins (org.dm.bundle and ampde)
```
Comment (WimCrols) [inline CHANGELOG.md]: It's important that the order stays chronologically. Can you put it back?

Comment (kpiot123) [reply CHANGELOG.md]: Isn't it chronologically now? I thought the ticket numbers were ordered form low to high.

Comment (Ranecdev) [reply CHANGELOG.md]: > Isn't it chronologically now? I thought the ticket numbers were ordered form low to high.

@kpiot123 I assume it is chronologically ordered based on implementation date and not on ticket creation date.

Comment (kpiot123) [reply CHANGELOG.md]: I don't think that is very practical if you try to read the changelog.

Comment (WimCrols) [reply CHANGELOG.md]: Chronological ordered on commit date. That's how all our changelog are. They mirror the git log.

Comment (Ranecdev) [reply CHANGELOG.md]: > Chronological ordered on commit date. That's how all our changelog are. They mirror the git log.
> Just put it back the way it was, guys :) There was never any reason to mess with the existing order anyway

Ok. The order was reverted in my last commit.

**Pull request merged**